scripts/commands/checkLinks.ts

```diff
@@ -117,6 +123,13 @@ const readArgs = (): Arguments => {
       description:
         "Should external links be checked? This slows down the script, but is useful to check.",
     })
+    .option("release-notes", {
```

Comment:
How about `qiskit-release-notes`? Since it only impacts that. The description would need to be changed too. 

Also the README needs updating.

Reply:
Done!